installer/conf/omsagent.d/oms_audits.xml

```diff
@@ -0,0 +1,1145 @@
+<data>
+ <audits BaselineId="OMS.Linux.1" BaseOrigId="1">
+  <audit
+    description="The nodev option should be enabled for all removable media."
+    msid="2.1"
+    cceid="CCE-3522-0"
+    severity="Important"
+    impact="An attacker could mount a special device (e.g. block or character device) via removable media"
+    remediation="Add the nodev option to the fourth field (mounting options) in /etc/fstab">
```

Comment:
The <remediation/> elements are spurious. Any reason not to remove their contents from the document?